src/mozillavpn.cpp

```diff
@@ -844,7 +847,7 @@ void MozillaVPN::errorHandle(ErrorHandler::ErrorType error) {
     } else {
       emit triggerGleanSample(GleanSample::authenticationFailure);
     }
-    setState(StateInitialize);
```

Comment:
I don't understand this change. The only result I see is that we trigger `triggerGleanSample`twice.
Can you tell me more why this fixes the android crash?